src/m_menu.c

```diff
@@ -3923,6 +3924,9 @@ setup_menu_t gen_settings2[] = { // General Settings screen2
   {"Brightmaps for Textures and Sprites", S_YESNO, m_null, M_X,
    G_Y3 + general_brightmaps*M_SPC, {"brightmaps"}},
 
+  {"Solid Status Bar Background", S_YESNO, m_null, M_X,
```

Comment:
I may need some help with this description from native speakers. @kraflab @mikeday0 Is it clear what is meant or do you have a better wording?

Reply:
Thank you!

Reply:
I think this is fine as is, but part of me wants something like "Smart Status Bar Background" to give a hint that it's going to analyze the status bar colors and pick something that matches. Cool feature btw!

Reply:
"Smart" sounds a bit too bold IMHO, it's still solid colors after all. But thanks for your input!